README.md

```diff
@@ -2,6 +2,15 @@
 
 A simple Calculator App built with HTML, CSS, and JavaScript. It also has a Dark Mode.
 
+This web based calculator is a project that helps anyone get an introduction to javascript with a basic web application. it is very barebones, but the simplicity helps make it easier to understand. 
+
+How to use:
+
+To use this calculator, just go to this site in your web browser. zxcodes.github.io/Calculator. If you are running this locally, you can insall NodeJS and run the program that way.
```

Comment:
You don't need Node JS as we're not using any server to server the ```index.html``` file. You can just clone the repo and run the ```index.html``` file and everything works.